homeassistant/components/renault/services.yaml

```diff
@@ -0,0 +1,76 @@
+ac_start:
+  description: Start A/C on vehicle.
+  fields:
+    vehicle:
+      name: Vehicle
+      description: The vehicle to send the command to.
+      required: true
+      selector:
+        device:
+          integration: renault
+    temperature:
+      description: Target A/C temperature in °C.
+      example: "21"
```

Comment:
Can we define selectors for the other parameters too?

https://www.home-assistant.io/docs/blueprint/selectors/

Reply:
I looked at this but I am unsure how to implement.
It seems that `min` and `max` are compulsory... should I just make a random call on what I think are valid values, and what a good `step` would be?
Also, there doesn't seem to be a `datetime` picker (only `time`).

Reply:
If number is too narrow we can use text eg. Make a best effort attempt.

Reply:
I have done `ac_start`, `ac_cancel` and `charge_start`. However I do not see how I can implement it for `charge_set_schedules`.
The `schedules` schema is quite complex!

Reply:
Maybe use object selector?

Reply:
I hadn't seen that selector in the list. Should be ok now.